components/UnorderedList/src/index.scss

```diff
@@ -22,3 +22,28 @@
 .denhaag-unordered-list--lower-alpha {
   list-style-type: var(--denhaag-unordered-list-lower-alpha-list-style-type);
 }
+
+.denhaag-unordered-list ul {
```

Comment:
Why do we need `ul` in this selector? Isn't `denhaag-unordered-list` a class name on the `ul`?

Reply:
> If for what reason, the nested `<ul>` doesn't have class="denhaag-unordered-list" this still will have the same css properties as .denhaag-unordered-list

Yeah thats exactly what I think you do not always want. I am wondering if we should have a wrapper component where such behaviour is added, maybe even for a lot more rich-text components without a class name. A bit like the [html-content component of Utrecht](https://github.com/nl-design-system/utrecht/blob/main/components/html-content/css/index.scss) 

Would also help quite a lot for unexpected behaviour of whitespace between components that is helpful and expected in the context of the (wordpress) website, but not necessarily within the context of other applications.

Reply:
> this is meant for a nested <ul> without the class="denhaag-unordered-list".
> For fallback purpose.

I don't really see who would benefit from this currently. Doesn't WordPress output class names for nested list items? The Gutenberg blocks appear to add those class names, from the looks of it.

Reply:
@Yolijn @Robbert I removed the fallback css now